### PR TITLE
13 coadds not saving

### PR DESCRIPTION
**Pull request by laldoroty**

### CODE REVIEW

File: sfft_and_animate.py [hunk 1]
```diff
@@ -177,12 +168,35 @@ def sfft_and_animate(oid,band):
...
-        sci_skysub_path = sky_subtract(band=band,pointing=pointing,sca=sca)
-        sci_imalign_path = imalign(ref_4k,sci_skysub_path,force=True)
-
         print('REF_4K_PATH')
         print(ref_4k)
 
+        # Retrieve and rotate PSFs for PSF coaddition.
+        template_psf_list = []
+        for i, row in enumerate(out_tab):
+            psf = get_imsim_psf(RA,DEC,out_tab['filter'][i],out_tab['pointing'][i],
```
Comment (wmwv) [inline sfft_and_animate.py:177]: enumerating and then using the index seems a little odd.  I think you want something simpler like:
```
for row in out_tab:
    psf = get_imsim_psf(RA, Dec, row["filter"], row["pointing"], row["sca"])
    template_psf_list.append(psf)
```

**Review by wmwv — CHANGES_REQUESTED** [review]
Suggestion for a cleaner iteration loop.

File: sfft_and_animate.py (hunk 1 above)
Comment (laldoroty) [reply sfft_and_animate.py:177]: Oops, this is an artifact of a brain fart. Thanks for catching.

**Comment by laldoroty** [comment]
Requested change pushed.

**Review by wmwv — APPROVED** [review]
LGTM

**Pull request merged**